tensorflow_addons/losses/python/lifted_test.py

```diff
@@ -19,10 +19,9 @@
 from __future__ import print_function
 
 import numpy as np
+import tensorflow as tf
 
-from tensorflow.python.framework import constant_op
-from tensorflow.python.framework import test_util
-from tensorflow.python.platform import test
+from tensorflow.python.framework import test_util as tf_test_util
```

Comment:
Ditto-- maybe it would be wise to come up with a list of APIs you had to pull from lower-level libs, and we can discuss whether they should be made available from the main TF namespace.

Reply:
Sounds good, after we compile the list should we submit them one by one as issues on TF core? Or as a batch somewhere?

Reply:
Compile and file a single issue to TF core. Tag me in it, and I'll figure out what we can do.

Reply:
Will do.